### DISCUSSION TITLE
Layer Object in v6?

### DISCUSSION THREAD

**Discussion by lance-lfx** [fabricjs/fabric.js#10041] [General]
I am in the process of developing a layer manager for a drawing tool using fabricjs in v6. In the latest v6 update and the Group Rewrite, I have noticed that a Layer object was mentioned and included in Group Rewrite but it never made it into the v6 update. Is there a reason for this omission and is this a feature that we should be expecting shortly? Just wanted to make sure that I am not duplicating code and classes if there is already an implementation in place. Thanks.

**Comment by asturur** [comment]
The reason was scope-creep mainly. Too many feature that were bound to interact nicely and work and make sense.
I think interactive group and the layour manager should allow you to make your layer object as an extension and fabric does not have to support it directly. Let me know how it goes
